python/cuxfilter/dashboard.py

```diff
@@ -110,14 +110,15 @@ class DashBoard:
     def __init__(
         self,
         charts=[],
-        data=None,
+        dataframe=None,
         layout=single_feature,
         theme=light,
         title="Dashboard",
         data_size_widget=True,
         warnings=False,
     ):
-        self._data = data
+        self._cuxfilter_df = dataframe
+        self._data = self._cuxfilter_df.data
```

Comment:
I guess this is because existing code already references `self._data` in quite a few places? Typically storing references to attributes on other objects is a little suspect. Could the query methods here be moved (and then delegated to) `cuxfilter.DataFrame`?

Reply:
So the `cuxfilter.Dashboard` class would only have a reference to `cuxfilter.DataFrame` object, and any references to `cuxfilter.DataFrame.data` in the dashboard class would be accessed using `self._cuxfilter_df.data`?

Reply:
I was actually suggesting trying to move accesses themselves out of this class. But a closer look seems like maybe that is not so simple. So then going through `self._cuxfilter_df.data` consistently would be a slight improve though, IMO

Reply:
Got it, I'm on it

Reply:
@bryevdv this is now implemented - data is consistently accessed throughout via cuxfilter.Dashboard._cuxfilter_df.data. I'm open to further refactoring this to completely move accesses out of the Dashboard class, may be a separate PR.